src/main.c

```diff
@@ -1475,7 +1475,8 @@ static void fork_server(int fd, conn_t *l, unsigned char no_fork) {
      * expected.
      */
 
-    exit(0);
+    pr_session_disconnect(NULL, PR_SESS_DISCONNECT_CONFIG_ACL,
+      "Blocked by <Limit LOGIN>");
```

Comment:
This is the fix for ensuring that we properly increment the "connection refused" `mod_snmp` counter, among other things.